data_structures/src/radon_report.rs

```diff
@@ -407,5 +407,9 @@ mod tests {
         ];
 
         assert_eq!(metadata.liars, expected);
+
+        // empty vec does not modify anything
```

Comment:
Please elaborate on this. Maybe something like this? (I don't know if I understood correctly)
```suggestion
        // An empty vector of new liars should cause no changes to the existing list of of liars 
```